layouts/partials/pagelayout.html

```diff
@@ -27,6 +27,10 @@ <h1 class="primary-color-dark f2">
         </section>
       </div>
 
+      <div class="nested-list-reset nested-links">
+        {{/* pagination.html: https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/template_embedded.go#L117 */}}
+        {{ template "_internal/pagination.html" .pagination_data }}
```

Comment:
- I initially tried using `.context` but it did not work with the following error

>  failed to render pages: render of "page" failed: execute of template failed at <partial "pagelayout.html" (dict "context" . "section_to_display" $section_to_display)>: error calling partial: execute of template failed at <$page.Paginator.TotalPages>: error calling TotalPages: runtime error: invalid memory address or nil pointer dereference 

- Am not too familiar with the inner workings thus added a separate key to the `dict` and it worked

So if there is a cleaner way do let me know and will be happy to change and confirm how it goes